docs/yum.md

```diff
@@ -7,7 +7,7 @@ This document describes how to install Percona Server for PostgreSQL from Percon
 If you intend to install Percona Distribution for PostgreSQL on Red Hat Enterprise Linux v8, disable the ``postgresql``  and ``llvm-toolset``modules:
 
 ```{.bash data-prompt="$"}
-$ sudo dnf module disable postgresql llvm-toolset
+$ sudo dnf module disable postgresql llvm-toolset rust-toolset
```

Comment:
I wouldn't recommend to disable rust. Also in future releases we will try to avoid disabling llvm-toolset